fdbserver/masterserver.actor.cpp

```diff
@@ -1250,6 +1257,17 @@ ACTOR Future<Void> serveLiveCommittedVersion(Reference<MasterData> self) {
 				++self->reportLiveCommittedVersionRequests;
 				req.reply.send(Void());
 			}
+			when(GetTlogPrevCommitVersionRequest req =
+			         waitNext(self->myInterface.getTlogPrevCommitVersion.getFuture())) {
+				GetTlogPrevCommitVersionReply reply;
+				for (Tag tag : req.tagSet) {
+					// TraceEvent("Received GetTlogPrevCommitVersionRequest").detail("Tag", tag.toString());
+					if (self->ssVersionVector.hasVersion(tag)) {
```

Comment:
Should we assert this condition?

Reply:
I assume this is dummy code - we can't do a lookup in "MasterData::ssVersionVector" to find the TPCV. We will need to introduce a new version vector in MasterData to hold the PCVs oft logs and the proxy will need to do a lookup in that version vector - I assume this will come in a follow up PR. Also, tlogs are identified by UUIDs?

Reply:
tlogs will be identified by a location int in this PR. I shall add a separate version vector for the TPCV